aws/src/main/java/org/apache/iceberg/aws/glue/LakeFormationCatalogProperties.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aws.glue;
+
+public class LakeFormationCatalogProperties {
+  private LakeFormationCatalogProperties() {
+  }
+
+  public static final String LAKE_FORMATION_AUTHORIZED_COLUMNS_KEY = "___lake_formation_authorized_columns___";
```

Comment:
As commented before, we can first remove column-level support in this PR to be more focused